lib/openshift/project.rb

```diff
@@ -150,6 +150,11 @@ def get_pods(by:, **get_opts)
     end
     alias_method :pods, :get_pods
 
+    def get_jobs(by:, **get_opts)
+      Job.list(user: by, project: self, **get_opts)
+    end
+    alias_method :jobs, :get_jobs
```

Comment:
where is this method used? If not used, then better not add it.

If we need it added, then `get_jobs` is not needed. Keeping some `get_something` methods for historic reasons only. They don't look good.

Reply:
It's used in step `a job appears with labels` and used in [feature](https://github.com/openshift/cucushift/blob/536d14aef3434e5d1bc9179c90866007fe280893/web/admin-console/cronjobs.feature#L45)

Reply:
also `get_jobs` is removed from `alias_method`